runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc

```diff
@@ -164,6 +164,13 @@ void FullyConnectedLayer::run()
 void FullyConnectedLayer::prepare()
 {
 #ifdef USE_RUY_GEMV
+  // The only fc hybrid will use ruy kernel
```

Comment:
What about leaving a TODO comment that this is a workaround and it should check if the weight is const.

Reply:
It should also check if the input is dynamic.